test/indexer/data/f0731ab4-6b80-4eed-97c9-4984de81a47c.metadata.json

```diff
@@ -187,12 +187,13 @@
         }
     },
     "analysis_file_0.json": {
-        "describedBy": "https://schema.humancellatlas.org/type/file/6.0.0/analysis_file",
+        "describedBy": "https://schema.staging.data.humancellatlas.org/type/file/6.3.0/analysis_file",
         "file_core": {
             "file_name": "merged-cell-metrics.csv.gz",
             "format": "unknown"
         },
         "schema_type": "file",
+        "matrix_cell_count": 100,
```

Comment:
In this case I'd be open to adding a canned bundle rather than modifying an old one.

Can we use the canned staging area plugin to can that bundle?